aws/resource_aws_cloudtrail.go

```diff
@@ -99,7 +99,7 @@ func resourceAwsCloudTrailCreate(d *schema.ResourceData, meta interface{}) error
 	if v, ok := d.GetOk("cloud_watch_logs_role_arn"); ok {
 		input.CloudWatchLogsRoleArn = aws.String(v.(string))
 	}
-	if v, ok := d.GetOk("include_global_service_events"); ok {
+	if v, ok := d.GetOkExists("include_global_service_events"); ok {
```

Comment:
Not sure this change needs to be part of this work. Can you confirm @atsushi-ishibashi?

Reply:
@Ninir Sorry, I checked out from #2817 
I'll remove this commit!